apps/src/javalab/Playground.js

```diff
@@ -162,10 +176,23 @@ export default class Playground {
     return document.getElementById('playground-background');
   }
 
+  getAudioElement() {
+    return document.getElementById('playground-audio');
+  }
+
+  resetAudioElement() {
```

Comment:
Do we need to also call `stop()` on the audio element or does replacing the `src` take care of it?

Reply:
Replacing the src takes care of it, but couldn't hurt I guess :)